### ISSUE TITLE
Performance of Waymo pretrained model

### ISSUE THREAD

**Issue by deeptibhegde** [cxy1997/3D_adapt_auto_driving#5]
I find that the performance of the waymo.pth model is quite lower than that mentioned in the paper. The bev/3D performance on NuScenes in the Easy category is 31.1754 / 14.4220 compared to the 55.5 / 21.6 in the paper. I see similar lower performance when evaluating on KITTI and Waymo.

**Comment by MartinHahner** [comment]
> I see similar lower performance when evaluating on KITTI and Waymo.

https://github.com/open-mmlab/OpenPCDet/issues/599

**Comment by mileyan** [comment]
Hi @deeptibhegde @MartinHahner,
We have updated the wrong pointrcnn code. I will update it soon. The original pointrcnn dataloader only works well for KITTI. We have to change the sampling method when using it on the other datasets.

**Comment by mileyan** [comment]
Hi @deeptibhegde and @MartinHahner ,
31.1754 / 14.4220 is correct. You can check table S6 in the appendix. You can get the table 2 result by setting `--metric new`. Please let me know if you have any other question.
